src/styles/index.scss

```diff
@@ -3 +1,30 @@
+@import '/src/moyo_heder';
+@import '/src/cards';
+@import '/src/main';
+
+@media screen and (max-width: 1024px) {
+  .nav-link:hover {
+    color: $main-bg-color;
+  }
+
+  .header {
+    width: 924px;
+    margin: 0 auto;
+  }
+
+  body {
+    width: 1024px;
+  }
```

Comment:
Setting a fixed width to the `body` element is not a common practice and can lead to horizontal scrollbars on smaller screens. Consider using `max-width` instead of `width` to limit the maximum width while keeping the layout responsive.